config/locales/en.yml

```diff
@@ -41,6 +41,7 @@ en:
     new:
       alert: "Already logged in"
       email: "Email"
+      logged_in: "You must be logged in"
```

Comment:
A suggestion to change the key to be `require_login:`, which is more descriptive.

Reply:
Also what do you think about nesting the flash messages like so:
```
new:
  flash:
    alert:
    require_login:
```

Reply:
The renaming makes sense and yes I can see logic behind the nesting, the 'new' list is getting long!
Done in sha: 08e222e